app/src/main/java/echoserver/App.java

```diff
@@ -1,14 +1,11 @@
-/*
- * This Java source file was generated by the Gradle 'init' task.
- */
 package echoserver;
 
 public class App {
-    public String getGreeting() {
-        return "Hello World!";
-    }
+  public String getGreeting() {
+    return "Hello World!";
+  }
 
-    public static void main(String[] args) {
-        System.out.println(new App().getGreeting());
-    }
+  public static void main(String[] args) {
+    System.out.println(new App().getGreeting());
```

Comment:
Creating the class instance inside of the same class doesn't feel right. And we usually do not create the instance of the class with the main method in Java. This class is responsible for launching the app. I understand that this is just a temporary solution for showing that app is running and tests are working. You don't need to change it, but there are possible refactoring suggestions: 
- move `getGreeting()` to the different class, create an instance of this class in main and call the method. 
- make this method `static` and call it directly from main.

Reply:
This was the default class created by the `gradle init` command when I initialized the Gradle project but I understand the issues you mentioned & appreciate that you brought them up! Already have modified that file for the next ticket and will make sure I follow these suggestions for all subsequent classes